Kubernetes/windows/debug/networkhealth.ps1

```diff
@@ -198,6 +200,15 @@ class NetworkTroubleshooter {
        
         ConvertTo-Html -Body "$heading $problemsDetectedHtml $issuesCheckedHtml $endpointData $remoteEndpointData $networkData $lbData $nodeData" -Head $Header | Out-File TroubleshootingReport.html
     }
+    
+    [void] LogErrors() {
+        
+        if ($this.NetworkStatus -eq [TestStatus]::Failed) {
+            $problemsDetected = $this.FailureSet | Select-Object Problem
+            $message = "{0} {1} {2}" -f (Get-Date).ToString(),$(hostname), ($problemsDetected | Format-Table | Out-String)

```

Comment:
let's not format it to a table and just do a single line instead.

Reply:
problemsDetected is an aray. You need to iterate over each problem and print it to a new line